client/src/components/deploy/DeployPage.js

```diff
@@ -44,7 +43,7 @@ const DeployPage = () => {
   const handleDeploy = () => {
     const accessToken = localStorage.getItem('GH_access_token');
     // call backend endpoint to deploy and give the access token
-    dispatch(deployToGHPages(teamId, accessToken, linkedHandle));
+    dispatch(deployToGHPages(teamId, accessToken));
```

Comment:
Do we not intend to send twitter handle information during deploy anymore?

Reply:
Yes, we do. But I refractor a bit and put twitter handle inside the teamInfo object.

Reply:
yep, missed that somehow my bad